src/main/java/core/basesyntax/strategy/ActivitiesShop.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitOperation;
+
+public interface ActivitiesShop {
+    void calculation(FruitOperation inputFruit);
```

Comment:
```suggestion
    void calculate(FruitOperation inputFruit);
```
використовуй дієслова для неймінгу методів